Tests/Fixtures/Generated/Colors/swift3-context-defaults-publicAccess.swift

```diff
@@ -49,7 +49,7 @@ public extension Color {
 }
 // swiftlint:enable operator_usage_whitespace
 
-public extension Color {
+internal extension Color {
```

Comment:
I think this one can be kept `public` though.

In multiple places in the documentation, we say that the end user has the choice when they want to access a color using our constants, to either use `ColorName.articleBody.color` or `UIColor(named: ColorName.articleBody)`. It's up to them to use `.color` property (which is public in your example) to create the color, or to create it using `UIColor(named: )`, whichever they prefer. So I think we should still allow this one to be `public` and not force it as `internal`

Reply:
Especially when you have a flat list of colors (no namespace and nested enums), you'd be able to do `UIColor(named: .articleBody)` which some people might prefer over `ColorName.articleBody.color` :wink:

Reply:
Ok, that does make sense. I've made the changes in fdf42d2. 👍